root/static/js/canto-modules.js

```diff
@@ -204,6 +204,10 @@ function isSingleLocusDiploid(genotype) {
   return Object.keys(diploidNames).length === 1;
 }
 
+function isWildTypeGenotype(genotype) {
+  return genotype.genotype_identifier.indexOf('wild-type-genotype') !== -1
+}
+
```

Comment:
I think this might be a more robust:

```suggestion
function isWildTypeGenotype(genotype) {
  return genotype.alleles.length == 0;
}
```

Reply:
Thanks. I suspected there would be a better way to test for wild type genotypes. I've committed these changes.